daos/MessageDao.ts

```diff
@@ -43,7 +43,7 @@ export default class MessageDao implements MessageDaoI {
      */
     findAllMessagesReceivedByUser = async (uid: string): Promise<Message[]> =>
```

Comment:
Doing Postman testing now and this does not work now that we have changed the message concept.
It was one of the ones that we said we don't really need anymore so for now I think it's fine but just putting this comment here as a reminder to either fix or remove it before we merge